examples/with-next-app/app/page.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
```

Comment:
fwiw, i would avoid client components at the root and try to get it as close to the leaves as possible so conumers can still leverage server components if they want.

two options:
- compose `ConnectButton` into a separate component tagged as a client component,
- [add the `banner` option to esbuild config](https://github.com/wagmi-dev/wagmi/blob/main/packages/react/tsup.config.ts#L8-L10)

Reply:
Ah, was wondering how to prevent esbuild from dropping that `use client;` tag. Thanks!